osu.Server.Spectator/Hubs/Metadata/MetadataHub.cs

```diff
@@ -68,7 +68,6 @@ public override async Task OnConnectedAsync()
                 usage.Item = new MetadataClientState(Context.ConnectionId, Context.GetUserId(), versionHash);
 
                 await logLogin(usage);
-                await broadcastUserPresenceUpdate(usage.Item.UserId, usage.Item.ToUserPresence());
```

Comment:
Just to confirm intent, this is deleted because it's basically conveying nothing, correct? I'm not really sure why I put that there in the first place but I guess that makes sense.

Reply:
Yeah, it should never fire anything.